### PR TITLE
refactor: update feed card styles for web

### PR DESCRIPTION
**Pull request by enesozturk**
# Why

There were some UI issues with the feed items on both web and mobile web states.

- https://linear.app/showtime/issue/SHOW2-815/nft-card-ui-bug
- https://linear.app/showtime/issue/SHOW2-877/nft-action-buttons-goes-under-bottom-navigation-on-web-mobile

# How

Refactored styles and handled some calculations for the web.  The bottom navigator height is returning 0 on the web, needed to define it hard coded also. 

# Test Plan

Checking styles on each platform.

### CODE REVIEW

File: packages/app/components/swipe-list.tsx [hunk 1]
```diff
@@ -90,9 +94,14 @@ export const SwipeList = ({
   const { height: safeAreaFrameHeight } = useSafeAreaFrame();
   const { height: windowHeight, width: windowWidth } = useWindowDimensions();
 
+  console.log("paddings", windowHeight, headerHeight, bottomPadding);
```
Comment (intergalacticspacehighway) [inline packages/app/components/swipe-list.tsx]: Log here

**Review by intergalacticspacehighway — APPROVED** [review]
LGTM ❇️

File: packages/app/components/swipe-list.tsx (hunk 1 above)
Comment (enesozturk) [reply packages/app/components/swipe-list.tsx]: omg

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

**Pull request merged**